assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -64,28 +64,15 @@ const EditControls = styled.div`
   justify-content: space-between;
   padding: 0;
   transition: opacity ease-in-out 300ms;
-  background: ${({ theme }) => theme.DEPRECATED_THEME.cardItem.previewOverlay};
-  border-radius: ${({ theme }) =>
-    theme.DEPRECATED_THEME.storyPreview.borderRadius}px;
+  background: ${({ theme }) => theme.colors.opacity.white16};
```

Comment:
if there's ever a dark mode, should we make this a specific bg value?

Reply:
Great point, will update!